app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionDetailsFragment.kt

```diff
@@ -0,0 +1,93 @@
+package org.wikipedia.suggestededits
+
+import android.os.Bundle
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.GONE
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.fragment_contribution_diff_detail.*
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.history.HistoryEntry
+import org.wikipedia.json.GsonUnmarshaller
+import org.wikipedia.page.PageActivity
+import org.wikipedia.page.PageTitle
+import org.wikipedia.suggestededits.Contribution.Companion.EDIT_TYPE_ARTICLE_DESCRIPTION
+import org.wikipedia.suggestededits.Contribution.Companion.EDIT_TYPE_IMAGE_CAPTION
+import org.wikipedia.suggestededits.SuggestedEditsContributionDetailsActivity.Companion.EXTRA_SOURCE_CONTRIBUTION
+import org.wikipedia.util.DateUtil
+import org.wikipedia.util.GradientUtil
+import org.wikipedia.views.ViewUtil
+
+class SuggestedEditsContributionDetailsFragment : Fragment() {
+    var contribution: Contribution? = null
```

Comment:
Would it be better if changes it to `lateinit var contribution: Contribution` so that we can remove all the `!!` mark after each `contribution`